src/Auth0.php

```diff
@@ -306,31 +312,16 @@ public function __construct(array $config)
             $this->skipUserinfo = $config['skip_userinfo'];
         }
 
-        // If a token algorithm is passed, make sure it's a specific string.
-        if (! empty($config['id_token_alg'])) {
-            if (! in_array( $config['id_token_alg'], ['HS256', 'RS256'] )) {
-                throw new CoreException('Invalid id_token_alg; must be "HS256" or "RS256"');
-            }
-
-            $this->idTokenAlg = $config['id_token_alg'];
-        }
-
-        // If a token audience is passed, make sure it's an array.
```

Comment:
Removing the option to set a custom audience. The audience should be the Client ID which we already have.